### DISCUSSION TITLE
Row and Column  number

### DISCUSSION THREAD

**Discussion by itsui** [vzhd1701/gridplayer#86] [Ideas]
Would be good to be able to choose the number of Row and Columns.
I have a lot of vertical videos (filmed on a smarthphone) and Gridplayer wastes a ot of space putting them in multiple rows,
Distributing them in Columns would be much more efficient in this case.

**Comment by vzhd1701** [comment]
You can set grid size via Grid -> Size. See https://github.com/vzhd1701/gridplayer/issues/50.
